main.nf

```diff
@@ -0,0 +1,32 @@
+// Ensure DSL2
+nextflow.enable.dsl = 2
+
+//test params
+params.config = "/agora-data-tools/test_config.yaml"
+
+//runs test config for Agora
+process AGORA_DATA_RUN {
+
+    debug true
```

Comment:
**Nit (_i.e._ [optional](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.)):** For this, perhaps add another profile as follows so it can easily be toggled on and off. 

```nextflow
profiles {

    docker {
        ...
    }

    test {
        ...
    }

    debug {
        process {
            debug = true
        }
    }

}
```